### DISCUSSION TITLE
OTP test numbers similar to firebase

### DISCUSSION THREAD

**Discussion by sameh-amwal** [supabase/supabase#6891] [Feature Requests]
Is there a way to have test numbers for phone Auth similar to the way firebase provides test numbers where the OTP is always 123456?

**Comment by anngbaum** [comment]
+1 -- and for a bit of context, we're exclusively using Twilio + OTP for authentication. We're trying to figure out a workaround in order to submit to the iOS App Store for review (since Apple requires a static login + password in order to review! 🙄)

**Reply by sameh-amwal** [reply]
Exactly same situation!!

**Comment by alexreyes** [comment]
+1 for reasons listed above
